tracing-subscriber/tests/layer_filters/main.rs

```diff
@@ -2,33 +2,34 @@
 mod boxed;
 mod downcast_raw;
 mod filter_scopes;
+mod option;
 mod per_event;
 mod targets;
 mod trees;
 mod vec;
 
 use tracing::{level_filters::LevelFilter, Level};
-use tracing_mock::{event, layer, span, subscriber};
+use tracing_mock::{event, expect, layer, subscriber};
 use tracing_subscriber::{filter, prelude::*, Layer};
 
 #[test]
-fn basic_layer_filters() {
+fn basic_subscriber_filters() {
```

Comment:
can we change this to use the v0.1.x names? e.g. `basic_layer_filters`, as it was previously?